reference_implementations/aws/offline/Readme.md

```diff
@@ -0,0 +1,172 @@
+## How to Create a Key Pair for AWS:
```

Comment:
Add a title and a brief description about this README, and maybe the system diagram as well. See the GCP one for an example: https://github.com/VectorInstitute/ai-deployment-bootcamp/tree/main/reference_implementations/gcp/architectures/offline#readme

Reply:
Added.